sourcegraph/sourcegraph
# app: Wrap outbound links on product pages

For links in Sourcegraph App that go to About, wrap the URLs with the analytics parameters.

The params are added using `addSourcegraphAppOutboundUrlParameters()`

## Test plan

- Start Sourcegraph App mode
- Visit the Batch Changes and Code Insights pages
- Check the outbound URLs to About have params added
- Start in non-Sourcegraph-App mode
- Check that the URLs do not have these params added in non-Sourcegraph-App mode

## App preview:

- [Web](https://sg-web-marek-wrap-outbound-links-1.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
